staging/src/k8s.io/client-go/tools/clientcmd/client_config.go

```diff
@@ -314,6 +316,41 @@ func canIdentifyUser(config restclient.Config) bool {
 		config.ExecProvider != nil
 }
 
+// cleanANSIEscapeCodes takes an arbitrary string and ensures that there are no
+// ANSI escape sequences that could put the terminal in a weird state (e.g.,
+// "\e[1m" bolds text)
+func cleanANSIEscapeCodes(s string) string {
```

Comment:
Is this tested?

Reply:
I believe this was addressed in https://github.com/kubernetes/kubernetes/pull/91305#discussion_r445629883, but let me know if not.